changes/next/contact_blobid

```diff
@@ -0,0 +1,13 @@
+Description:
+
+Always return 'blobId' in /set{create.update} and  /get responses.
```

Comment:
I'd state the JMAP object type that this change applies to. e.g. "Always return 'blobId' in in JMAP Contact /set...."